testing/MLDB-1336-builtin-checks.py

```diff
@@ -11,9 +11,6 @@ def test_sequence(self):
         with self.assertRaisesRegexp(mldb_wrapper.ResponseException, "Executing builtin function exp: Can't convert value 'a' of type 'ASCII_STRING' to double") as re:
             query = "SELECT exp('a')"
             mldb.query(query)
-        with self.assertRaisesRegexp(mldb_wrapper.ResponseException, "Executing builtin function ln: the argument of the ln function must be strictly positive") as re:
-            query = "SELECT ln(-1)"
```

Comment:
Can we keep the test to check that it DOESNT crash and that it returns a consistent value?